src/main/java/uk/gov/companieshouse/api/accounts/request/AccountsNoteConverter.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.companieshouse.api.accounts.request;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.enumeration.AccountType;
+import uk.gov.companieshouse.api.accounts.enumeration.AccountingNoteType;
+import uk.gov.companieshouse.api.accounts.enumeration.NoteType;
+import uk.gov.companieshouse.api.accounts.exception.InvalidPathParameterException;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+
+@Component
+public class AccountsNoteConverter {
+
+    private static final EnumMap<AccountType, EnumMap<NoteType, AccountingNoteType>> ACCOUNTS_NOTE_MAP = new EnumMap<>(AccountType.class);
+
+    AccountsNoteConverter() {
```

Comment:
Just curious , what is the reason behind adding a constructor with default access ?

Reply:
Just overriding the default constructor to build up the class member `EnumMap` 👍

Reply:
yeah got that but should it be private AccountsNoteConverter() ? If we are going for Singelton class ?

Reply:
Making it private would be mean the default no-args constructor for the class wouldn't be available, and Spring would fail to inject this as a dependency